winui/Rating/Overview.md

```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
```

Comment:
The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service, such as movie applications, products, and more.

Reply:
changed the content as mentioned above.